pkg/conn/basedb_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conn
+
+import (
+	"github.com/DATA-DOG/go-sqlmock"
+	. "github.com/pingcap/check"
+	tcontext "github.com/pingcap/dm/pkg/context"
```

Comment:
add a blank line.

Reply:
Done in [here](https://github.com/pingcap/dm/pull/266/files#diff-8dd3ef1da219ede5563625a2fc6ceb7cR19)